src/app/modules/group/components/group-log-view/group-log-view.component.html

```diff
@@ -35,15 +35,21 @@
           pTemplate="body"
```

Comment:
> The breadcrumb should be clickable and navigate to the hovered item with the selected path (and no attempt)

The breadcrumb is a link, but no path is given. For instance: "https://dev.algorea.org/branch/feature/path-suggestion-overlay/en/#/activities/by-id/369645559920184960"

Reply:
Fixed

Reply:
Hmm, shall I create pipe for that?

Reply:
I think constructing the url (using item-router) in the component is enough, you can map over breadcrumbs and add a new property. I was thinking [here](https://github.com/France-ioi/AlgoreaFrontend/pull/985/files#diff-6664e60575412437d7dd0a3a95ee8bb818194e7f0736f4ed3e1c6bfe6c3358c5R26)

Reply:
Fixed